packages/desktop-client/src/components/settings/Export.tsx

```diff
@@ -27,14 +28,14 @@ export function ExportBudget() {
     if ('error' in response) {
       setError(response.error);
       setIsLoading(false);
-      console.log('Export error code:', response.error);
+      console.log(t('Export error code:'), response.error);
```

Comment:
```suggestion
      console.log('Export error code:', response.error);
```

Shouldn't need to translate console logs